packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -193,6 +230,9 @@ export const chatMessageSlotClassNames: ChatMessageSlotClassNames = {
   author: `${chatMessageClassName}__author`,
   badge: `${chatMessageClassName}__badge`,
   bar: `${chatMessageClassName}__bar`,
+  body: `${chatMessageClassName}__body`,
```

Comment:
What's the difference between this and `compactBody`? Could we merge them?

Reply:
> When I test the demo (https://fluentuipr.z22.web.core.windows.net/pull/23974/react-northstar/maximize/chat-example-comfy-v2), the timestamp is rendered below the message bubble and cannot be hovered: ![image](https://user-images.githubusercontent.com/2564094/180318886-7f57e454-a9ed-45b2-926b-29b7b463b33f.png)

Yeah, sorry about that. I must have messed something up while refactoring some code. Addressing it now.

Reply:
Possibly, we can talk today.